policykit/templates/base.html

```diff
@@ -11,6 +11,7 @@
         <title>PolicyKit</title>
         <!-- Styles -->
         <link rel="stylesheet" href={% static "policyengine/css/output.css" %}>
+        <link rel="stylesheet" href={% static "policyengine/css/login.css" %}>
```

Comment:
This was a change made to master but I don't think is working in the new frontend branch. The styling on the homepage seems to be broken currently, but works in the login page.